### PR TITLE
DO-1776 Make method for finding city of registration

### PR DESCRIPTION
**Pull request by EmmaSamoilova**

### CODE REVIEW

File: business_register/converter/declaration.py [hunk 1]
```diff
@@ -96,7 +98,35 @@ def find_country(self, property_country_data):
...
     def find_city(self, registration_data):
-        pass
+        city, region, district = self.normalized_registration_data(registration_data)
+        city_of_registration = None
+        ratu_region = RatuRegion.objects.filter(name=region).first()
+        if not ratu_region:
+            logger.error(f'cannot find region {region}')
+        else:
+            if region == 'київ' or region == 'севастополь':
+                city_of_registration = ratu_region
```
Comment (IegorSoboliev) [inline business_register/converter/declaration.py:108]: There will be ValueError. Maybe, good idea is to store both cities as City object

File: business_register/converter/declaration.py [hunk 2]
```diff
@@ -96,7 +98,35 @@ def find_country(self, property_country_data):
...
+            elif region and city and district:
+                ratu_district = RatuDistrict.objects.filter(name=district, region_id=ratu_region.id).first()
+                if not ratu_district:
+                    logger.error(f'cannot find district {district}')
+                else:
+                    city_of_registration = RatuCity.objects.filter(
+                        name=city, region_id=ratu_region.id, district_id=ratu_district.id
+                    ).first()
+            elif region and city:
```
Comment (IegorSoboliev) [inline business_register/converter/declaration.py:117]: You can retrieve an object with None district field

**Review by IegorSoboliev — CHANGES_REQUESTED** [review]

File: business_register/converter/declaration.py [hunk 3]
```diff
@@ -96,7 +98,44 @@ def find_country(self, property_country_data):
...
     # TODO: extract registration data
     def find_city(self, registration_data):
-        pass
+        city, region, district = self.normalized_registration_data(registration_data)
+        city_of_registration = ''
+        ratu_region = RatuRegion.objects.filter(name=region).first()
+        if not ratu_region:
+            logger.error(f'cannot find region {region}')
+        else:
+            if region == city:
```
Comment (IegorSoboliev) [inline business_register/converter/declaration.py:107]: Do you mean Kyiv and Sevastopol? Are you sure there will be no other matches?

File: business_register/converter/declaration.py (hunk 2 above)
Comment (IegorSoboliev) [reply business_register/converter/declaration.py:117]: No, I mean you can filter lile this 
```
RatuCity.objects.filter(
                        name=city, region_id=ratu_region.id, district_id=ratu_district.id
                    ).first()
```
even with district_id = Null

**Review by IegorSoboliev — APPROVED** [review]

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**